src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -130,7 +130,7 @@ function UserEnrollmentForm({ className }) {
                   <span aria-hidden>
                     I agree to GetSmarter&apos;s{' '}
                     <Hyperlink
-                      destination="https://www.getsmarter.com/terms-and-conditions-for-students"
+                      destination={process.env.GETSMARTER_STUDENT_TC_URL}
```

Comment:
by adding the environment variables to frontend-platform's config via `mergeConfig` in `src/index.jsx`, we would want to grab the config values from frontend-platform rather than reading directly from `process.env` here, e.g.:

```jsx
import { getConfig } from '@edx/frontend-platform/config';

function UserEnrollmentForm() {
  const config = getConfig();
  return <Hyperlink destination={config.GETSMARTER_STUDENT_TC_URL} ... />
}
```

Reply:
Oh, but by doing that, you'll end up needing to mock `getConfig` in the test!

Reply:
There's some prior art for mocking `getConfig` in the `EnterpriseInvitePage.test.jsx` file 😄